app/views/map/_leaflet.html.erb

```diff
@@ -5,10 +5,21 @@
 <div class="leaflet-map" id="map<%= unique_id %>"></div>
 <% if defined? people %><p><i><small>Share your own location on <a href='/profile'>your profile</a>.</small></i></p><% end %>
 <script>
-  var map = L.map('map<%= unique_id %>').setView([<%= lat %>,<%= lon %>], <%= lat.to_s.length.to_i %> + 6);
-  L.tileLayer("//a.tiles.mapbox.com/v3/jywarren.map-lmrwb2em/{z}/{x}/{y}.png").addTo(map);
-
-  <% items.each do |item| %>
-    L.marker([<%= item.lat %>, <%= item.lon %>]).addTo(map).bindPopup("<a href='<%= item.path %>'><%= item.title %></a>");
-  <% end %>
+  var map<%= unique_id %> = L.map('map<%= unique_id %>').on('load', onMapLoad).setView([<%= lat %>,<%= lon %>], <%= lat.to_s.length.to_i %> + 6);
+  L.tileLayer("//a.tiles.mapbox.com/v3/jywarren.map-lmrwb2em/{z}/{x}/{y}.png").addTo(map<%= unique_id %>);
+  function onMapLoad(e){
+	 var lat = <%= lat %> ;
+	 var lon = <%= lon %> ;
+	 var s = lat+","+lon ;
+ 	$.getJSON("/api/srch/locations?srchString="+s , function(data){
+	 			if (!!data.items){
```

Comment:
Oh, this is looking super, but do you think we could do really consistent two-space indentations here so it's cleanly formatted? Sorry to ask, but it does make a big difference over the long run. Thank you!

Reply:
Yes @jywarren , i agree and understand the importance of indentation . Doing the changes now , thanks 😄 💯 !